### PR TITLE
Expose the Koji API on the standard https port

### PR DESCRIPTION
**Pull request by teg**
See individual commits for details.

### CODE REVIEW

**Review by Gundersanne — DISMISSED** [review]

**Review by larskarlitski — CHANGES_REQUESTED** [review]
I agree that it should use standard ports, but unfortunately the tests need to be adjusted (predicted by @gicmo):
```[2020-09-16T17:56:30.176Z] Error: cannot listen on the TCP port: listen tcp4 :443: bind: address already in use
```
Port 443 of one of the containers (kojid?) is forwarded to localhost.

File: internal/upload/koji/run-koji-container.sh [hunk 1]
```diff
@@ -81,7 +81,7 @@ koji_start() {
   ${CONTAINER_RUNTIME} run -d --name org.osbuild.koji.koji --network org.osbuild.koji \
     -v "${SHARE_DIR}:/share:z" \
     -p 80:80 \
-    -p 443:443 \
+    -p 443:4343 \
```
Comment (ondrejbudai) [inline internal/upload/koji/run-koji-container.sh:84]: This is backwards, see https://docs.docker.com/config/containers/container-networking/

```suggestion
    -p 4343:443 \
```

**Review by ondrejbudai — CHANGES_REQUESTED** [review]

File: internal/upload/koji/run-koji-container.sh (hunk 1 above)
Comment (gicmo) [inline internal/upload/koji/run-koji-container.sh:84]: I think it is a bit weird to have koji exposed on standard `80` but then the corresponding standard https port (`443`) be composer. Maybe also move koji to `8080` or something?

**Comment by gicmo** [comment]
I like the idea! It will need a lot of changes in the koji-plugin test because they all assume that koji runs on the default http/https port.

File: internal/upload/koji/run-koji-container.sh (hunk 1 above)
Comment (teg) [reply internal/upload/koji/run-koji-container.sh]: Changed that too.

Comment (teg) [reply internal/upload/koji/run-koji-container.sh]: Fixed!

**Review by ondrejbudai — APPROVED** [review]
Looks good to me. Let's wait for the CI though.

**Comment by teg** [comment]
All koji tests passed, the others are unrelated.

**Pull request merged**